src/content/developers/docs/scaling/sidechains/index.md

```diff
@@ -7,29 +7,70 @@ incomplete: true
 sidebarDepth: 3
 ---
 
-A sidechain is a separate blockchain which runs in parallel to Ethereum Mainnet and operates independently. It has its own [consensus algorithm](/developers/docs/consensus-mechanisms/) (e.g. [proof-of-authority](https://wikipedia.org/wiki/Proof_of_authority), [Delegated proof-of-stake](https://en.bitcoinwiki.org/wiki/DPoS), [Byzantine fault tolerance](https://decrypt.co/resources/byzantine-fault-tolerance-what-is-it-explained)). It is connected to Mainnet by a two-way bridge.
+A sidechain is a separate blockchain which runs in parallel to Ethereum; it is connected to Mainnet by a two-way bridge. Sidechains have separate [consensus algorithms](/developers/docs/consensus-mechanisms/), which are often designed for faster processing times. Using a sidechain involves trade-offs, though, as they do not inherit Ethereum's security properties, unlike [layer 2 scaling solutions](/layer-2/). 
 
-What makes a sidechain particularly exciting is that the chain works the same as the main Ethereum chain because it's based on [the EVM](/developers/docs/evm/). It doesn't use Ethereum, it is Ethereum. This means if you want to use your [dapp](/developers/docs/dapps/) on a sidechain, it's just a matter of deploying your code to this sidechain. It looks, feels, and acts just like Mainnet – you write contracts in Solidity, and interact with the chain via the Web3 API.
+## How do sidechains work? {#how-do-sidechains-work}
 
-## Prerequisites {#prerequisites}
+### Consensus algorithms {#consensus-algorithms}
```

Comment:
Before this h3, I would into the "How do sidechains work?" section with a loose overview that these are basically separate blockchains, with their own histories, and their own parameters that make them unique. 

We could then lead into how consensus algorithms are one of the ways they can be unique.

Would also suggest adding information about some other common parameters that are changed, like block gas limits and slot/block times.

Reply:
More on that last point, we could potentially use this to expand and have a section that mentions how these parameter adjustments impact the decentralization of the network. In short, things like fast block times and high gas limits will give fast/cheap transactions, but will make it significantly harder to operate a node, which can easily make a network/chain with a dangerously small cohort of validators. 

We mention how validators can more easily collude on these chains, this could be the precursor to why that happens.

Reply:
I added a section to explain block parameters. LMK your thoughts on it.

Reply:
Looks good! Thanks!